src/main/resources/META-INF/plugin.xml

```diff
@@ -29,7 +29,7 @@
     <depends>com.intellij.properties</depends>
     <depends>org.jetbrains.idea.maven</depends>
     <depends>com.intellij.gradle</depends>
-    <idea-version since-build="222" until-build="222.*"/>
+    <idea-version since-build="222" until-build="223"/>
```

Comment:
Do we have to use `223.*` to include all versions of 223? Looking at https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html seems like yes?

Reply:
👍🏼 Seems like a safe bet.